test/extensions/filters/http/common/fuzz/uber_per_filter.cc

```diff
@@ -129,7 +133,7 @@ void UberFilterFuzzer::cleanFuzzedConfig(absl::string_view filter_name,
   } else if (name == HttpFilterNames::get().Squash) {
     cleanAttachmentTemplate(message);
   } else if (name == HttpFilterNames::get().Tap) {
-    // TapDS oneof field not implemented.
+    // TapDS oneof field and OutputSinkType StreamingGrpc not implemented
```

Comment:
I can't add a comment at the line (because of GitHub UI/diff), but where `cleanFuzzedConfig` is called in `uber_filter.cc`, suggest:
`// Clean-up config with filter-specific logic.` -> `// Clean-up config with filter-specific logic, before it runs through validations`